CHANGELOG.md

```diff
@@ -121,6 +121,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - "GET /api/test/monkey" endpoint. #2269
 - "GET /api/test/log" endpoint. #2269
 - Node Map from Security Report. #2334
+- "Accessible From" and "Services" from the preview pane in the map on the Island. #2430
```

Comment:
Island doesn't say much to a user unfamiliar with our architecture. Instead we should say something like "from the preview pane in the map page". Page means UI, it's clear what's been done